### PR TITLE
use constant for unknown owner keyword

### PR DESCRIPTION
**Pull request by nickpell**
https://github.com/dimagi/commcare-hq/pull/20962#discussion_r197780685

### CODE REVIEW

File: corehq/apps/export/const.py [hunk 1]
```diff
@@ -142,3 +142,6 @@ class SharingOption(object):
         EXPORT_ONLY,
         EDIT_AND_EXPORT,
     )
+
+
+UNKNOWN_EXPORT_OWNER = 'unknown'
```
Comment (proteusvacuum) [inline corehq/apps/export/const.py]: Should this be translated since it is being shown to users?

Comment (proteusvacuum) [reply corehq/apps/export/const.py]: why not just `ugettext_lazy` the constant, then you can remove that `if` statement?

Comment (nickpell) [reply corehq/apps/export/const.py]: There's a formatting different (see the ```div```) between user names and "Unknown"

Comment (nickpell) [reply corehq/apps/export/const.py]: @proteusvacuum I let a lot of time go by before addressing this - let me know if you want me to ping my new code buddy for review.

**Review by proteusvacuum — APPROVED** [review]

**Pull request merged**